Readme.md

```diff
@@ -119,7 +119,9 @@ a) Directory structure of our current Templates
 
    ./
 
-	|__conf: For all configurations and credential files
+	|__conf: For all configurations 
+
+	|__env For credential files
```

Comment:
Are we creating new directory "env" and keeping .env file there?

Reply:
No. Do you say , no need to mention about the ".env" in structure ? env is not as folder , I was mentioning it as file

Reply:
Here your representation shows at env. Please remove it from structure

Reply:
already did! Latest change will not show env in structure